plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/ndt_infra.py

```diff
@@ -525,7 +525,8 @@ def create_topoconfig_file(links_info_dict, ndt_file_path, patterns,
                 topoconfig_file.write("%s,%s,%s,%s,%s,%s\n" % (port_guid, start_port,
                             peer_port_guid, peer_port,host_type,port_state))
     ndt_file.close()
-    return True, "", failed_lable_conversion
+    msg = "Failed to resolve port label to port number. Check log file." if failed_lable_conversion else ""
```

Comment:
message is not informative - user cannot tell which port label has failed...

Reply:
it is in log file
could be many